package.json

```diff
@@ -8,6 +8,7 @@
     "@svgr/webpack": "4.1.0",
     "@types/hellojs": "1.16.2",
     "@uifabric/styling": "6.46.0",
+    "adaptivecards": "^1.1.0",
```

Comment:
+1.

This way we always have exact version of dependecies running on all machines. Prevents the it works on my machine syndrom. :-)

Reply:
@jobala  I think having a `yarn.lock` or `package-lock` have the exact version, so when you run `npm i` it will try to install the latest changes but if there is a `lock file` in the dir then it will install the exact version specified in the `lock file` unless someone runs an upgrade command.

Reply:
This is now resolved to align to convention.